lldb/source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -765,12 +766,13 @@ bool SwiftUserExpression::Parse(DiagnosticManager &diagnostic_manager,
         std::static_pointer_cast<SwiftASTContextForExpressions>(swift_ast_ctx);
 
   if (!m_swift_ast_ctx)
-    return error("could not create a Swift AST context");
+    return error("could not initialize Swift compiler",
+                 "run 'swift-healthcheck' for more details");
```

Comment:
These are user facing right? Running swift-healthcheck won't produce anything useful to the user, right? I'd change it to run swift-healthcheck and file a bug with the file.

Reply:
It is user-facing, but it contains, for example compiler errors that users are expected to be familiar with.